chapter_crashcourse/ndarray.md

```diff
@@ -176,6 +176,6 @@ x
 
 ## 总结
 
-ndarray模块提供一系列多维数组操作函数。所有函数列表可以参见[NDArray API文档](https://mxnet.incubator.apache.org/api/python/ndarray.html)。
+ndarray模块提供一系列多维数组操作函数。所有函数列表可以参见[NDArray API文档](https://mxnet.incubator.apache.org/api/python/index.html#ndarray-api)。
```

Comment:
would it be:
https://mxnet.incubator.apache.org/api/python/ndarray/ndarray.html

Reply:
yeah that looks better